master-docker-nonstandard-2/master.cfg

```diff
@@ -178,10 +180,11 @@ addWorker(
 addWorker(
     "apexis-bbw",
     3,
-    "-ubuntu-2204-jepsen-mysql",
-    "quay.io/mariadb-foundation/bb-worker:ubuntu22.04-jepsen-mysql",
+    "-ubuntu-2204-jepsen-mariadb",
+    "quay.io/mariadb-foundation/bb-worker:ubuntu22.04-jepsen-mariadb",
     jobs=5,
     save_packages=False,
+    volumes=["jepsen:/home/buildbot/.lein", "m2:/home/buildbot/.m2"],
```

Comment:
It's not clear for me why we're mounting a host path

Reply:
volumes removed, it was trying to keep the files lein downloads